### ISSUE TITLE
Improve --target option

### ISSUE THREAD

**Issue by jacintoArias** [demosense/demorepo#11]
I find confusing the available options for specifying the target projects. I think that `--target`, `--all-targets` and `--ci-tool` could be unified.

My proposal is that:

- `demorepo run` Runs the pipeline for *EVERY* project
- `demorepo run --targets <project list>` Runs the pipeline for every project specified in list. List must be specified as whitespaced separated list, in order to allow `grep` commands (see next section). Also blob patterns should be allowed in the future.

A new command `demorepo ci-changeset` (or something), that prints a whitespace separated list of the modified targets (by last green commit and --ci--tool specified). This list could be piped into previous command or cached in a variable.

`demorepo run --target $(demorepo ci-changeset)`
```
TARGET=`demorepo ci-changeset`
demorepo run --target $TARGET
```

**Comment by jcozar87** [comment]
I am not so convinced about the command name, but I agree with you.

Respect to using multiple targets, the doc was incorrect: currently multiple target projects are provided separated by blank spaces. Doc modified in commit #13.

**Comment by jacintoArias** [comment]
Closed by #21
